src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java

```diff
@@ -12,21 +12,42 @@
 
 /** Metadata for a blob that sends to Snowflake as part of the register blob request */
 class BlobMetadata {
+  public static final long DEFAULT_BLOB_LATENCY = -1;
```

Comment:
is there a particular reason of using -1, could we just use Long and pass NULL? That's what we did for most of the values, for example `first_insert_time_in_ms`

Reply:
Using NULL Long instead, thanks for the suggestion